projects/m1/018-roulette-payouts/js/index.js

```diff
@@ -0,0 +1,86 @@
+function generateRouletteElements( specialValues, lowestValue, highestValue ){
+    let elementsList = [];
+
+    elementsList = elementsList.concat( specialValues );

```

Comment:
It might be a good idea to use: **Spread Operator**
`const elementList = [...specialValues]`